src/schema.mjs

```diff
@@ -125,6 +125,47 @@ export const jsonrpc = {
   ],
 };
 
+export const ipfs = {
+  type: "object",
+  properties: {
+    type: {
+      type: "string",
+      enum: ["ipfs"],
+    },
+    commissioner: {
+      type: "string",
+    },
+    version: {
+      type: "string",
+    },
+    options: {
+      type: "object",
+      properties: {
+        timeout: {
+          $comment: "temporal unit is milliseconds",
+          type: "integer",
+        },
+        url: { type: "string" },
```

Comment:
If this is possible, then at this stage there would be a change to filter for invalid IPFS CIDs using a regular expression, it would conveniently also fix: https://github.com/neume-network/extraction-worker/issues/33

Reply:
I think there is more to CID and we can't use regular expression. I have take care of this at the extraction-worker level in https://github.com/neume-network/extraction-worker/commit/0949f14d7b4ec2b2f2dc69817ddd165da9f32bf9.

Reply:
yes, I'm of that impression too